src/SDKs/Peering/Management.Peering/generated/EdgeRpClient.cs

```diff
@@ -0,0 +1,410 @@
+// <auto-generated>
```

Comment:
Was the `generated` directory auto-generated

Reply:
yes